### ISSUE TITLE
Appveyor Failure on Python 3.7

### ISSUE THREAD

**Issue by Olllom** [MolSSI/cookiecutter-cms#83]
`create_conda_env.py` fails for an appveyor build on python 3.7.
https://ci.appveyor.com/project/Olllom/pyworkdir/builds/26263346/job/iw36fc15qq5kh5dt

Python 3.6 build is OK
https://ci.appveyor.com/project/Olllom/pyworkdir/builds/26263346/job/45wft6qedcwoao05

The issue is known 
https://github.com/conda/conda-build/issues/3220
and can be fixed easily
https://github.com/matplotlib/matplotlib/pull/14649/files

The appveyor error message is.
```
python devtools\\scripts\\create_conda_env.py -n=test -p=%PYTHON_VERSION% devtools\\conda-envs\\test_env.yaml
173Traceback (most recent call last):
174  File "C:\Miniconda37-x64\Scripts\conda-env-script.py", line 5, in <module>
175    from conda_env.cli.main import main
176  File "C:\Miniconda37-x64\lib\site-packages\conda_env\cli\main.py", line 39, in <module>
177    from . import main_create
178  File "C:\Miniconda37-x64\lib\site-packages\conda_env\cli\main_create.py", line 12, in <module>
179    from conda.cli import install as cli_install
180  File "C:\Miniconda37-x64\lib\site-packages\conda\cli\install.py", line 19, in <module>
181    from ..core.index import calculate_channel_urls, get_index
182  File "C:\Miniconda37-x64\lib\site-packages\conda\core\index.py", line 9, in <module>
183    from .package_cache_data import PackageCacheData
184  File "C:\Miniconda37-x64\lib\site-packages\conda\core\package_cache_data.py", line 15, in <module>
185    from conda_package_handling.api import InvalidArchiveError
186  File "C:\Miniconda37-x64\lib\site-packages\conda_package_handling\api.py", line 3, in <module>
187    from libarchive.exception import ArchiveError as _LibarchiveArchiveError
188  File "C:\Miniconda37-x64\lib\site-packages\libarchive\__init__.py", line 1, in <module>
189    from .entry import ArchiveEntry
190  File "C:\Miniconda37-x64\lib\site-packages\libarchive\entry.py", line 6, in <module>
191    from . import ffi
192  File "C:\Miniconda37-x64\lib\site-packages\libarchive\ffi.py", line 27, in <module>
193    libarchive = ctypes.cdll.LoadLibrary(libarchive_path)
194  File "C:\Miniconda37-x64\lib\ctypes\__init__.py", line 434, in LoadLibrary
195    return self._dlltype(name)
196  File "C:\Miniconda37-x64\lib\ctypes\__init__.py", line 356, in __init__
197    self._handle = _dlopen(self._name, mode)
198TypeError: LoadLibrary() argument 1 must be str, not None
199CONDA ENV NAME  test
200PYTHON VERSION  3.7
201CONDA FILE NAME devtools\\conda-envs\\test_env.yaml
202CONDA PATH      C:\Miniconda37-x64\Scripts\conda.EXE
203activate test
204Could not find conda environment: test
```

**Comment by dgasmith** [comment]
Interesting, can you make a PR to patch this?

**Comment by dgasmith** [comment]
Closed in #84 + we not longer use AppVeyor :D.
